app/Http/Controllers/IndexController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Models\Client;
+use App\Models\Client_contact;
+
+
+class IndexController extends Controller
+{
+    public function index(Request $request)
+    {
+        $domain = $request->getHost();
+        $client = Client::where('base_url', $domain)->first();
```

Comment:
This where should be in a middleware which then store client or client_id to a session